src/transaction.js

```diff
@@ -4,9 +4,15 @@ var bufferutils = require('./bufferutils')
 var opcodes = require('./opcodes')
 var typeforce = require('typeforce')
 var types = require('./types')
+var BufferWriter = require('./bufferwriter')
+
+var ADVANCED_TRANSACTION_MARKER = 0
+var ADVANCED_TRANSACTION_FLAG = 1
 
 function Transaction () {
   this.version = 1
+  this.marker = null
+  this.flag = null
```

Comment:
Are these ever going to change?
My understanding was the `0` marker was to allow for easy detection in serialization?